packages/nvidia-k8s-device-plugin/1100-k8s-device-plugin-all-device-patch.patch

```diff
@@ -0,0 +1,36 @@
+From ae5521585da70ff06ae66c9b02613cf85b5c3c0a Mon Sep 17 00:00:00 2001
```

Comment:
Posting here for the lack of a better place to add this comment: you have a typo in your commit message